Assets/Scripts/Interactable/Door/InteractableNetworkManager.cs

```diff
@@ -14,17 +14,17 @@ public void RegisterInteractable(IInteractable interactable)
         Interactables.Add(interactable);
     }
 
-    public void ProcessDoorForAllClients(int id)
+    public void ProcessInteractableForAllClients(int id)
     {
         var doorPanel = Interactables.Find(i => i.GetId() == id);
```

Comment:
Where this is for other interactables, should this variable renamed so it doesn't specifically refer to doorPanels? :)

Reply:
This is minor though so I am happy to approve this for now and for this to be fixed up in a later PR.